src/js/components/Notification/Notification.js

```diff
@@ -72,8 +80,10 @@ const Notification = ({ message, onClose, status, title, toast }) => {
         role="log"
         modal={false}
         onEsc={onClose}
+        id={id}
         responsive
         plain
+        position={position === 'top-right' ? 'top-right' : 'top'}
```

Comment:
Rather than default here, I think we should default in the initial declaration and take whatever the caller passes in. So, "bottom-right" would work. From a grommet perspective, I think we should be this flexible.

Reply:
I was going back and forth with whether or not to just allow all of the properties. I'll reconfigure